modules/ossm-deploying-multiple-control-planes.adoc

```diff
@@ -0,0 +1,196 @@
+// Module included in the following assemblies:
+// install/ossm-deploying-multiple-service-meshes-on-single-cluster.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-deploying-multiple-control-planes_{context}"]
+= Deploying multiple control planes
+:context: ossm-deploying-multiple-control-planes
+
+To configure a cluster to host two control planes, set up separate {istio} resources with unique names in independent {istio} system namespaces. Assign a unique revision name to each {istio} resource to identify the control planes, workloads, or namespaces it manages. Apply these revision names using injection or `istio.io/rev` labels to specify which control plane injects the sidecar proxy into application pods.
+
+Each `{istio}` resource must also configure discovery selectors to specify which namespaces the {istio} control plane observes. Only namespaces with labels that match the configured discovery selectors can join the mesh. Additionally, discovery selectors determine which control plane creates the `istio-ca-root-cert` config map in each namespace, which is used to encrypt traffic between services with mutual TLS within each mesh.
+
+When adding an additional {istio} control plane to a cluster with an existing control plane, ensure that the existing `{istio}` instance has discovery selectors configured to avoid overlapping with the new control plane.
+
+This procedure may be extended to support more than two control planes. The maximum number of service meshes in a single cluster depends on the available cluster resources.
+
+[NOTE]
+====
+There is only one `IstioCNI` resource shared by all control planes in the cluster, and it must be updated independently.
+====
+
+.Prerequisites
+* The {SMProduct} operator is installed.
+* An {istio} CNI resource is created.
+* The `istioctl` binary is installed on your localhost.
+
+[NOTE]
+====
+You can run the following command to check for existing `{istio}` instances:
+
+[source,terminal]
+----
+$ oc get istios
+----
+====
+
+.Procedure
+
+. Deploy the first control plane by using the following steps:
+
+.. Create the namespace for the first {istio} control plane called `istio-system-1`:
++
+[source,terminal]
+----
+$ oc new-project istio-system-1
+----
+
+.. Add the following label to the first namespace, which will be used with the Istio `discoverySelectors` field:
++
+[source,terminal]
+----
+$ oc label namespace istio-system-1 istio-discovery=mesh-1
+----
+
+.. Create a YAML file named `istio-1.yaml` with the name `mesh-1` and the discoverySelector `mesh-1`:
++
+.Example configuration
+[source,yaml,subs="attributes,verbatim"]
+----
+kind: Istio
+apiVersion: sailoperator.io/v1alpha1
+metadata:
+  name: mesh-1
+spec:
+  namespace: istio-system-1
+  values:
+    meshConfig:
+      discoverySelectors:
+        - matchLabels:
+            istio-discovery: mesh-1
```

Comment:
Is this a complete YAML? IF not, add:

[# ...](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#yaml-formatting)

Reply:
Apply throughout.